src/govuk/all.test.mjs

```diff
@@ -47,12 +47,12 @@ describe('GOV.UK Frontend', () => {
     it('exported Components have an init function', async () => {
       await goTo(page, '/')
 
-      var componentsWithoutInitFunctions = await page.evaluate(() => {
+      const componentsWithoutInitFunctions = await page.evaluate(() => {
```

Comment:
**question**: That doesn't feel JSDoc related, nor the change in `accordion.test.js`. Is it due to changes in the linting configuration?

Reply:
Yeah it's a side effect of https://github.com/alphagov/govuk-frontend/pull/3103/commits/bebed90b207efb5843374fb49643da353ec87495

The new JSDoc rules weren't being applied to test files because we'd included/excluded them incorrectly

(Side effect being that `'no-var': 'off'` now correctly applies to source but not tests)